tests/org.eclipse.ui.workbench.texteditor.tests/src/org/eclipse/ui/workbench/texteditor/tests/FindReplaceDialogTest.java

```diff
@@ -148,12 +151,16 @@ private void assertInitialConfiguration() {
 			assertFalse(regExCheckBox.getSelection());
 		}
 
-		void close() {
+		void closeAndRestore() {
 			restoreInitialConfiguration();
 			assertInitialConfiguration();
 			closeOperation.run();
```

Comment:
Just a nitpicky final comment. 
```suggestion
			close();
```

Reply:
Done in a767ca3